modules/http-impl/src/main/java/org/mule/services/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -72,8 +69,6 @@
 
   private static final Logger logger = LoggerFactory.getLogger(GrizzlyHttpClient.class);
 
-  private UriParameters uriParameters;
```

Comment:
Yes, this was extension specific (because host and port could support expression at the config level but we didn't want extra configs/connections being created). I've created the HttpExtensionClient with this and the authentication too.